File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/typescript/ADV-04b_C.ts
console.log("ADV-START");
const s = new Int32Array(1);
s[0] = -1;
const u = new Uint32Array(1);
u[0] = 1;
const cmp = s[0] < u[0];
console.log("OBS=CMP:" + cmp);
console.log("ADV-END");

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/cpp/ADV-04b_C.cpp
#include <cstdio>
#include <cstdint>

int main() {
    printf("ADV-START\n"); fflush(stdout);
    std::int32_t a = -1;
    std::uint32_t b = 1;
    bool c = a < b;
    printf("OBS=CMP:%s\n", c ? "true" : "false"); fflush(stdout);
    printf("ADV-END\n"); fflush(stdout);
    return 0;
}
